### PR TITLE
add myself to authors.md

### PR DESCRIPTION
**Pull request by tmakar**
## Basics

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which contains `_(my name)_`)
      **Please always add them to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - add a line in `doc/news/_preparation_next_release.md`
  - reformat the code with `scripts/dev/reformat-all`
  - make all unit tests pass
  - fix all memleaks
  - fix the CI itself (or rebase if already fixed)
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation (see [Documentation Guidelines](https://www.libelektra.org/devgettingstarted/documentation))
- [ ] I fixed all affected decisions (see [Decision Process](https://www.libelektra.org/decisions/decision-process))
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://www.libelektra.org/devgettingstarted/coding))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if everything is done** and no further pushes are planned by you.

### CODE REVIEW

File: doc/AUTHORS.md [hunk 1]
```diff
@@ -166,3 +166,11 @@ documentation, Python bindings for elektraMerge, copy-on-write functionality of
 - email: max@maxirlinger.at
 - GitHub user: [atmaxinger](https://github.com/atmaxinger)
 - devel/test on: Fedora, macOS
+
+## Tomislav Makar
+
+documentation, go-elektra (spec globbing use-case), rewrite spec plugin
```
Comment (markus2330) [inline doc/AUTHORS.md:172]: Please add link to go-elektra

```suggestion
spec plugin, go-elektra, web-ui
```

**Review by markus2330 — CHANGES_REQUESTED** [review]
Please also assign yourself to modules. Please clarify with @atmaxinger.

File: doc/AUTHORS.md (hunk 1 above)
Comment (tmakar) [reply doc/AUTHORS.md:172]: @markus2330 should I separate `go-elektra` and `web-ui` here, I thought of putting the web-ui (actually a web-api) into the `go-elektra` repo? What do you think?

**Comment by tmakar** [comment]
@atmaxinger What modules can I assign myself?

File: doc/AUTHORS.md (hunk 1 above)
Comment (tmakar) [reply doc/AUTHORS.md:172]: @markus2330 and I would call it `web-api` as it will be an api and no graphical user interface.

Comment (markus2330) [reply doc/AUTHORS.md]: You simply write in src/tools/elektrad/README.md etc. that you are the maintainer (infos/maintainer). Please use a header like we already have for plugins and bindings.

@atmaxinger please do the coordination about this and start with a PR that shows how to do it properly.

**Comment by atmaxinger** [comment]
@tmakar basically what you should do is what flo91 does in #4871. Add yourself as a maintainer to the README of the modules you need to touch in your thesis.

**Review by atmaxinger — CHANGES_REQUESTED** [review]
Please add the line

```
infos/maintainer = Tomislav Makar <tmakar23@gmail.com>
```

to the top of 

- src/tools/elektrad/README.md
- src/plugins/spec/README.md
- src/tools/webd/README.md

File: src/plugins/spec/README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+- infos/maintainer = Tomislav Makar <tmakar23@gmail.com>
```
[the remark this answers is not in the document]
Comment (tmakar) [reply src/plugins/spec/README.md:1]: ahh interesting, didn't know that it affects the build process. thanks!

[the remark this answers is not in the document]
Comment (atmaxinger) [reply src/plugins/spec/README.md]: Yes, it converts the infos section of the plugins into a `.c` file that is then included in the source of the plugins!

**Comment by atmaxinger** [comment]
Seems like you also need to update the man page [doc/man/man1/kdb-run-elektrad.1](https://github.com/ElektraInitiative/libelektra/pull/4866/checks?check_run_id=12064382266)

**Comment by tmakar** [comment]
> Seems like you also need to update the man page [doc/man/man1/kdb-run-elektrad.1](https://github.com/ElektraInitiative/libelektra/pull/4866/checks?check_run_id=12064382266)

Yes, I have read something about `ronn-ng` in the error logs.

**Review by atmaxinger — APPROVED** [review]
LGTM 👍

**Review by hannes99 — APPROVED** [review]
looks good

**Comment by tmakar** [comment]
@markus2330 What do you think? Can we merge? 😃

**Comment by atmaxinger** [comment]
Please rebase to resolve merge conflicts

**Pull request merged**